src/App.tsx

```diff
@@ -1,93 +1,49 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { TodosFilter } from './components/TodosFilter';
+import { Todo } from './types/todoTypes';
+import { Status } from './types/enumTypes';
+import { TodoContext } from './components/TodoContext';
+import { Header } from './components/Header';
+import { useLocalStorage } from './components/hooks/useLocalStorage';
 
 export const App: React.FC = () => {
-  return (
-    <div className="todoapp">
-      <header className="header">
-        <h1>todos</h1>
-
-        <form>
-          <input
-            type="text"
-            data-cy="createTodo"
-            className="new-todo"
-            placeholder="What needs to be done?"
-          />
-        </form>
-      </header>
-
-      <section className="main">
-        <input
-          type="checkbox"
-          id="toggle-all"
-          className="toggle-all"
-          data-cy="toggleAll"
-        />
-        <label htmlFor="toggle-all">Mark all as complete</label>
-
-        <ul className="todo-list" data-cy="todoList">
-          <li>
-            <div className="view">
-              <input type="checkbox" className="toggle" id="toggle-view" />
-              <label htmlFor="toggle-view">asdfghj</label>
-              <button type="button" className="destroy" data-cy="deleteTodo" />
-            </div>
-            <input type="text" className="edit" />
-          </li>
+  const [todos, setTodos] = useLocalStorage<Todo[]>('todos', []);
+  const [filterBy, setFilterBy] = useState('');
```

Comment:
```suggestion
  const [filterBy, setFilterBy] = useState(Status.All);
```
let's set Status.All by default